content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/body.skiptomaincontent.html

```diff
@@ -0,0 +1,26 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2020 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<div class="core-skip-to-main-content"
+     data-sly-use.page="com.adobe.cq.wcm.core.components.models.Page"
+     data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"
+     data-sly-test="${page.mainContentSelector}">
+    <a class="core-skip-to-main-content__link"
```

Comment:
I'd call it `cmp-page__skiptomaincontent-link`

Reply:
or `cmp-page__link--skiptomain`

Reply:
how about `cmp-page__skiptomain` and `cmp-page__skiptomain-link`?